receptorctl/pyproject.toml

```diff
@@ -2,5 +2,23 @@
 requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
 
+[project]
```

Comment:
FYI, I saw that using PEP 621 `[project]` table together with `setup.cfg` may result in rather surprising behaviors, so it's best to use just one place for the metadata.